enforcer/model.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Optional, Dict, Any, List
+
+from pydantic import BaseModel
+
+
+class PodOwner(BaseModel):
+    kind: str
+    name: str
+    namespace: str
+
+class RsOwner(BaseModel):
+    rs_name: str
+    namespace: str
+    owner_name: str
+    owner_kind: str
+    deletion_ts: Optional[float] = None
+
+class Resources(BaseModel):
+    request: float
+    limit: Optional[float]
+
+
+class ResourcesRecommendation(BaseModel):
```

Comment:
Is this essentially a ContainerRecommendation? If so, call it that. (The contrast with WorkloadRecommendation would help clarify what is going on.)

Reply:
fixed